webpack.config.js

```diff
@@ -34,40 +34,42 @@ module.exports = smp.wrap({
       },
       {
         test: /\.(png|jpe?g|gif|svg|ttf|woff2|woff|eot)$/i,
-        use: [{loader: 'file-loader',},],
+        use: [{loader: 'file-loader', },],
       },
       {
         test: /\.css$/i,
         use: [MiniCssExtractPlugin.loader, 'css-loader'],
       },
     ],
   },
-
   devServer: {
     contentBase: path.join(__dirname, 'dist'),
     compress: true,
-    port: 8000
+    port: 8000,
+    watchOptions: {
```

Comment:
Why do you need to change to polling mode? It is supposed to work out of the box: https://webpack.js.org/configuration/watch/#watchoptionspoll

Reply:
The automatic reload of the app doesn't work on my desktop computer. This fixed the issue.